### PR TITLE
PR for Profle,comments

### PR DESCRIPTION
**Pull request by suryxks**
features implemeted:

- follow user
- unfollow user
- add comment
- delete comment
- follow user suggestions
- [preview](https://socially-git-proflecomments-suryxks.vercel.app/)

### CODE REVIEW

**Review by ashishkatiyar2404 — APPROVED** [review]
Well structured code, looks good to me
You can merge and work on other functionality

File: src/pages/Profile.tsx [hunk 1]
```diff
@@ -18,18 +40,37 @@ export const Profile = () => {
...
+                        isCurrentUser ?
+                            <ButtonCta onClick={() => {dispatch(openEditModal())}}>Edit</ButtonCta> :
+                            isUserFollowed ? <ButtonOuLined onClick={unfollowHandler}>Unfollow</ButtonOuLined> :
+                                <ButtonCta onClick={followHandler}>Follow</ButtonCta>
+                    }
                     <UserBio>{profile?.bio}</UserBio>
                     <UserLink href={profile?.website} target='_blank'>{profile?.website}</UserLink>
                     <UserFollowWrapper>
                         <UserFowllowLink to='/'><UserFollowCount>{`${profile?.following.length} `}</UserFollowCount><span>Following </span></UserFowllowLink>
                         <UserFowllowLink to='/'><UserFollowCount>{`${profile?.followers.length} `}</UserFollowCount><span>Followers</span></UserFowllowLink>
```
Comment (Namrata-J) [inline src/pages/Profile.tsx:53]: its good that you are using optional chaining here

File: src/pages/index.ts [hunk 2]
```diff
@@ -1,7 +1,8 @@
+import { SinglePostPage } from './SinglePostPage';
 import { BookMarks } from './BookMarks';
 import { Profile } from './Profile';
 import { Home } from 'pages/Home';
 import { SignUp } from './SignUp';
 import { Login } from 'pages/Login';
 import { Explore } from 'pages/Explore';
-export {Explore,Login,SignUp,Home,Profile,BookMarks}
\ No newline at end of file
+export {Explore,Login,SignUp,Home,Profile,BookMarks,SinglePostPage}
```
Comment (Namrata-J) [inline src/pages/index.ts:8]: you can use export { explore } from '....' directly here, instead of first importing and then exporting

**Review by Namrata-J — APPROVED** [review]
rest of the code looks great to me!!

File: src/pages/index.ts (hunk 2 above)
Comment (suryxks) [reply src/pages/index.ts:8]: Thanks for pointing it will do from next time

**Pull request merged**